src/console/App.test.js

```diff
@@ -1,5 +1,5 @@
 import App from './App';
-import { wrapMockStore } from 'normandy/tests/mockStore';
+import { wrapMockStore } from 'console/tests/mockStore';
```

Comment:
I find that a bit confusing: importing `console/tests/` (which is aliased to `src/tests/`) from `src/console/`

Reply:
The `console` subfolder has been removed, so now we can trust that `console/...` is pointing to the root of the source files.